src/main/java/bio/terra/cli/apps/DockerAppsRunner.java

```diff
@@ -124,12 +122,33 @@ public void runToolCommand(
     }
     envVars.putAll(terraReferences);
 
-    buildDockerClient();
+    // mount the global context directory and the workspace directory to the container
+    //  e.g. global context dir on host = $HOME/.terra
+    //       global context dir on container = CONTAINER_HOME_DIR/.terra
+    //       workspace dir on host = $HOME/terra-workspaces/workspace123
+    //       workspace dir on container = CONTAINER_WORKSPACE_DIR
+    Path globalContextDirOnContainer =
+        GlobalContext.getGlobalContextDir(Path.of(CONTAINER_HOME_DIR));
+    Path workspaceDirOnContainer = Path.of(CONTAINER_WORKSPACE_DIR);
+
+    Map<String, File> bindMounts = new HashMap<>();
```

Comment:
optional style: keep the keys as `Path`s instead of `String`s to preserve some meaning until they're turned into `Volume`s.

Reply:
Good idea. Done.